iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/AbstractIdentifiableAdder.java

```diff
@@ -21,6 +23,8 @@
 
     private String name;
 
+    protected Properties properties;
```

Comment:
I'm not convinced that it's useful anymore. In the JsonImporter, you will read a classic Map<String,String>.
After creating the Identifiable instance, you can write something like that:
```
List<Extension> extensions = null;
Map<String, String> properties = null;
while (parser.nextToken() != JsonToken.END_OBJECT) {
    switch (parser.getCurrentName()) {
    ....
    case PROPERTIES:
        parser.nextToken();
        properties = readProperties(parser);
        break;
    case EXTENSIONS:
        parser.nextToken();
        extensions = readExtensions(parser);
        break;
    ...
    }
}
Identifiable identifiable = adder.add();
if (properties != null) {
    identifiable.getProperties().putAll(properties);
}
if (extensions != null) {
    identifiables.addExtensions(extensions); // This method should be added and you should provide a default implementation
}
```